File: src/main/java/com/sberbank/pfm/test/concordion/extensions/exam/db/DbResultRenderer.java
package com.sberbank.pfm.test.concordion.extensions.exam.db;

import com.sberbank.pfm.test.concordion.extensions.exam.rest.RestResultRenderer;
import org.concordion.api.listener.AssertEqualsListener;
import org.concordion.api.listener.AssertFalseListener;
import org.concordion.api.listener.AssertTrueListener;

public class DbResultRenderer extends RestResultRenderer
        implements AssertEqualsListener, AssertTrueListener, AssertFalseListener {
}

File: exam-db/src/main/java/io/github/adven27/concordion/extensions/exam/db/DbResultRenderer.kt
package io.github.adven27.concordion.extensions.exam.db

import io.github.adven27.concordion.extensions.exam.core.ExamResultRenderer
import org.concordion.api.listener.AssertEqualsListener
import org.concordion.api.listener.AssertFalseListener
import org.concordion.api.listener.AssertSuccessEvent
import org.concordion.api.listener.AssertTrueListener

class DbResultRenderer : ExamResultRenderer(), AssertEqualsListener, AssertTrueListener, AssertFalseListener {
    override fun successReported(event: AssertSuccessEvent) {
        event.element.addStyleClass("table-success").appendNonBreakingSpaceIfBlank()
    }
}
